src/plugins/intel_cpu/src/emitters/snippets/aarch64/utils.cpp

```diff
@@ -184,8 +189,23 @@ void push_ptrs_with_offsets_to_stack(dnnl::impl::cpu::aarch64::jit_generator* h,
     OV_CPU_JIT_EMITTER_ASSERT(mem_ptrs.size() == buffer_ids.size(), "mem_ptrs and buffer_ids size mismatch");
     OV_CPU_JIT_EMITTER_ASSERT(mem_ptrs.size() == stack_offsets.size(), "mem_ptrs and stack_offsets size mismatch");
 
-    // Store all pointers with offsets to their specific stack locations
+    // Fast path: pair-store original pointers when two consecutive entries do not need adjustment
+    std::vector<char> handled(mem_ptrs.size(), 0);
```

Comment:
`std::vector<bool>` looks as more aligned with the usage

Reply:
Done